include/GroupBox.h

```diff
@@ -50,6 +50,21 @@ class GroupBox : public QWidget, public BoolModelView
 		return m_led;
 	}
 
+	/**
+	 * @brief Returns whether the LED button is shown or not
+	 * 
+	 * @return true LED button is shown
+	 * @return false LED button is hidden
+	 */
+	bool ledButtonShown() const;
+
+	/**
+	 * @brief Sets if the LED check box is shown or not
+	 * 
+	 * @param value Set to true to show the LED check box or to false to hide it.
+	 */
+	void setLedButtonShown(bool value) const;
```

Comment:
Even it can be a `const` method, I don't think it should because it changes the state of the widget.

Reply:
Adjusted with commit 1ce2d116825.